examples/server/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+project(RealsenseExamplesServer)
+
+add_executable(rs-server rs-server.cpp ${CMAKE_SOURCE_DIR}/third-party/easyloggingpp/src/easylogging++.cc)
```

Comment:
Examples don't use ELPP directly - is there a specific need?

Reply:
Moved to the tools.